### PR TITLE
Mf change contributions

### PR DESCRIPTION
**Pull request by mfunk**
Updated README to move instructions for contributing to the contributing MD. Also updated instructions to use feature branch, rather than fork/clone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,10 +125,10 @@ Please note that some toolboxes are for ArcGIS Pro only, or ArcGIS Desktop 10.4
 
 ### Running Verification Tests
 
-[Running tests for solutions-geoprocessing-toolbox](.utils/test/Readme.md)
+- Follow instructions in [Running tests for solutions-geoprocessing-toolbox](.utils/test/Readme.md)
```
Comment (csmoore) [inline README.md]: missing "/"  `(./utils/test/Readme.md)`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+### Getting Changes from the repo
+The solutions-geoprocessing-toolbox repo changes without notice, so make sure you are getting the latest updates often.
+
+Using git command line check your *remote*:
+
+`> git remote -v`
+
+This command should return a list of remotes including:
+
+`Esri	https://github.com/Esri/solutions-geoprocessing-toolbox.git (fetch)`
```
Comment (csmoore) [inline CONTRIBUTING.md]: If you want to preserve multiline formatting use "```" before/after those lines:

**Comment by csmoore** [comment]
Reviewed. Just one bad link to fix. 

Another recommendation would be to create a "graphics" folder to put those root/top-level graphics in (ex. "CreateNewBranchDialogScreenshot.png") so there are not all in the repo root

File: CONTRIBUTING.md (hunk 2 above)
Comment (mfunk) [inline CONTRIBUTING.md]: That formatting doesn't seem to like carriage returns for multiple lines. It took booth lines and made them one. An extra CR made it worse. I'll leave it like that for now. :(

Comment (csmoore) [inline CONTRIBUTING.md]: Even when using multiline/block format? (i.e. "```") - https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting

**Comment by mfunk** [comment]
@csmoore I can't push and further fixes to this PR because commit 4a3190fdf25acde3500d99a4db2cba7dbead2457 caused a conflict. I made the fixes but have to put them in another CR.

**Comment by csmoore** [comment]
I have seen a few repos committing directly to dev - is this in error or new workflow?

**Comment by mfunk** [comment]
Closing this for a newer PR.

**Pull request closed without merging**